pages/Transactions/index.tsx

```diff
@@ -1,8 +1,56 @@
+import * as React from 'react';
```

Comment:
Avoid wildcard imports, in general, just import what you need.
```suggestion
import React, { useState } from 'react';
```